app/views/content/non-uk-teachers/return-to-england-after-teaching-overseas.md

```diff
@@ -6,7 +6,6 @@ description: |-
   Return to England after teaching outside the UK and bring your skills and experience back to an English classroom. Get support to make your transition easier.
 related_content:
     Salaries and benefits of teaching in England : "/salaries-and-benefits"
-    Returning to teaching with international experience : "/blog/returning-to-teaching-with-international-experience"
```

Comment:
GIven that we are not deleting this international returners page just yet, should we keep this link (and the blog itself) @philspray ?